src/app/modules/people/people.module.ts

```diff
@@ -41,4 +41,4 @@ import { SinglePageComponent } from './alumni/single-page/single-page.component'
     MarkdownModule.forRoot()
   ]
 })
-export class PeopleModule {}
+export class PeopleModule {}
```

Comment:
This file shouldn't be modified. You've removed a newline at the end. Please add it back.